components/header/avatar.tsx

```
@@ -0,0 +1,46 @@
+  sm: "h-8 w-8 text-xs",
+  md: "h-10 w-10 text-sm",
+  lg: "h-12 w-12 text-base",
+};
+
+export function Avatar({ src, alt = "", fallback, size = "sm", className }: AvatarProps) {
+  const [imageError, setImageError] = useState(false);
+  const dimensions = parseInt(sizeMap[size].split(" ")[0].replace("h-", "")) * 4; // Convert tailwind sizes to pixels
+
+  if (src && !imageError) {
+    return (
+      <div className={"relative rounded-full overflow-hidden" + sizeMap[size] + className}>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)